spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/cql/CassandraConnectorSpec.scala

```diff
@@ -115,6 +117,19 @@ class CassandraConnectorSpec extends SparkCassandraITFlatSpecBase {
     val conn2 = CassandraConnector(conf)
     conn2.withSessionDo { session => }
   }
+
+  it should "use compression when configured" in {
+    val host = EmbeddedCassandra.getHost(0).getHostAddress
+    val conf = new SparkConf(loadDefaults = false)
+      .set(CassandraConnectorConf.CassandraConnectionHostProperty, host)
+      .set(CassandraConnectorConf.CassandraConnectionCompressionProperty, "SNAPPY")
```

Comment:
Ok I see you are setting the compression option here to test it. It wasn't noted in the test at the bottom.